client_test.go

```diff
@@ -910,7 +910,7 @@ func TestAuthModelFromJson(t *testing.T) {
 
 			if test.expectedErr != "" {
 				c.Assert(err, qt.ErrorMatches, test.expectedErr)
-				c.Assert(model, qt.DeepEquals, []openfga.TypeDefinition(nil))
+				c.Assert(model, qt.IsNil)
```

Comment:
Is this line needed if we're testing the error?

Reply:
It's not required but I don't see much harm to having an extra assertion here. What do you think?